packages/amplify-graphql-schema-generator/src/ts-schema-generator/helpers.ts

```diff
@@ -147,19 +188,92 @@ export const createSchema = (schema: Schema, config?: DataSourceGenerateConfig):
     throw new Error('No valid tables found. Make sure at least one table has a primary key.');
   }
 
+  const engine = schema.getEngine().type;
+
+  const nullEnums = schema  // find null enums in all models
```

Comment:
naming nit: I think you mean "nullable enum fields" or rather than "null" enums? (The latter implies a concrete value assignment, as opposed to a field type constraint)

Reply:
I will update the naming to have more clear variable names to avoid confusion. NullEnumFields and RequiredEnumFields makes sense I will go ahead with these naming conventions to provide more readability and understanding